src/libraries/System.Memory/tests/ReadOnlySpan/Split.char.cs

```diff
@@ -0,0 +1,261 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Linq;
+using Xunit;
+using SpanSplitEnumerator = System.MemoryExtensions.SpanSplitEnumerator<char>;
+
+namespace System.SpanTests
+{
+    public static partial class ReadOnlySpanTests
```

Comment:
Can you make the existing string.Split test cases work for this as well, validating that the new Split produces the same splits as string.Split does?

Reply:
I may not be understanding completely, do you mean:
1. Find the test cases for `string.Split`
2. Move them into `tests/ReadOnlySpan/Split.char.cs`
or just add an Assert to the custom AssertEquals method in this file that does an additional `string.Split` to ensure the result of the `SpanSplitEnumerator` and `string.Split` align?

Reply:
> just add an Assert to the custom AssertEquals method in this file that does an additional string.Split to ensure the result of the SpanSplitEnumerator and string.Split align?

This :)

Reply:
Done